src/SamplesApp/SamplesApp.Wasm/Program.cs

```diff
@@ -11,6 +11,7 @@ public static void Main(string[] args)
 		{
 			// Required to allow for Puppeteer to select XAML elements in the HTML DOM.
 			Uno.UI.FeatureConfiguration.UIElement.AssignDOMXamlName = true;
+			Uno.UI.FeatureConfiguration.UIElement.AssignDOMXamlProperties = true;
```

Comment:
Perf issue at test time in CI (set debug only)

Reply:
:-)